packages/evm/test/enclave/Enclave.spec.ts

```diff
@@ -75,17 +66,16 @@ describe("Enclave", function () {
 
   describe("constructor / initialize()", function () {
     it("correctly sets owner", async function () {
-      // uses the fixture deployment with this.owner set as the owner
       const owner1 = await this.enclave.owner();
-      // create a new deployment with this.otherAccount as the owner
-      // note that this.owner is msg.sender in both cases
-      const enclave = await this.Enclave.deploy(this.otherAccount.address, this.otherAccount.address, this.maxDuration);
-      const owner2 = await enclave.owner();
+      const [, , , otherOwner, otherRegistry] = await ethers.getSigners();
```

Comment:
Should we just add this to the fixture, since it is repeated a few times already?